e2e/tests/upgrades/upgrade_test.go

```diff
@@ -55,6 +56,15 @@ type UpgradeTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+func (s *UpgradeTestSuite) SetupTest() {
+	channelOpts := s.TransferChannelOptions()
+	// TODO(chatton) hack to handle special case for the v8 to v8.1 upgrade test.
```

Comment:
maybe not the end of the world as it's a once off, I can create an issue for this but it would be nice to have a generalized way of specifying per channel options per test name. Kind of chicken and egg situation as it's happening in `SetupTest`